config/settings/common.py

```diff
@@ -290,6 +290,8 @@
 # Application Behaviors
 GITHUB_STATUS_UPDATES_ENABLED = env.bool("GITHUB_STATUS_UPDATES_ENABLED", True)
 METACI_FLOW_SUBCLASS_ENABLED = env.bool("METACI_FLOW_SUBCLASS_ENABLED", True)
+SCRATCH_ORG_RESERVE = env.int("METACI_SCRATCH_ORG_RESERVE", 10)
+# number of scratch orgs to leave available in the org.
```

Comment:
By default, this won't allow builds to run with a DE dev hub.

Reply:
good catch, lemme set a very low default in the app.json for free setup.